webpack.common.js

```diff
@@ -78,6 +78,13 @@ module.exports = {
     })
   ],
 
+  resolve: {
+    alias: {
+      '~': path.resolve(__dirname, 'src/assets'),
```

Comment:
Is this good practice? I don't know about the JavaScript world but tilde is usually used for home directory in the Linux/UNIX world so it does look strange to me.

Reply:
@vpetersson, how about `@`? Most of the JavaScript users tend to use the `@` sign.

Reply:
Sure. I don't know the common practices well enough. It just looked odd to me.